pyqtgraph/widgets/BusyCursor.py

```diff
@@ -1,35 +1,28 @@
-from ..Qt import QtGui, QtCore, QT_LIB
+# -*- coding: utf-8 -*-
+from contextlib import contextmanager
 
-__all__ = ['BusyCursor']
+from ..Qt import QtGui, QtCore
 
-class BusyCursor(object):
-    """Class for displaying a busy mouse cursor during long operations.
+__all__ = ["BusyCursor"]
+
+
+@contextmanager
+def BusyCursor():
```

Comment:
are the `()` necessary here?

Reply:
Yeah, since it's a function declaration, not a class any more. No parens here gives a SyntaxError.

Reply:
omg...i was _convinced_ this was a class ....

Reply:
🤦🏻‍♂️